docs/data/xrootd/install-standalone.md

```diff
@@ -1,2 +1,182 @@
+
+-   Ensure the host has [a supported operating system](/release/supported_platforms)
+-   Obtain root access to the host
+-   Prepare [the required Yum repositories](/common/yum)
+-   Install [CA certificates](/common/ca)
+
+Installing XRootD
+-----------------
+
+To install XRootD, run the following Yum command:
+
+``` console
+root@xrootd-server # yum install xrootd
+```
+
+Configuring XRootD
+------------------
+
+You will need to modify the file `/etc/xrootd/xrootd-server.cfg` that we will refer now on (unless explicitly said) as THE configuration file.
+A simple example of such a configuration is below with further customizations in the rest of the document.
+
+``` file
+xrd.port 1094
+all.role server
+# Allow all contents of /store to be served from this host
+all.export %RED%/store/ global %ENDCOLOR% readonly
+
+cms.allow host *
+# Logging verbosity
+xrootd.trace emsg login stall redirect
+ofs.trace -all
+xrd.trace conn
+cms.trace all
+
+all.adminpath /var/run/xrootd
+all.pidpath   /var/run/xrootd
+# This site name is only used for monitoring purposes (ex: T2_US_UCSD or UCSD)
+all.sitename   %RED%YOUR SITE NAME%ENDCOLOR%
+
+xrd.report     xrootd.t2.ucsd.edu:9931,desire.physics.ucsd.edu:9931 every 30s all sync
+xrootd.monitor all auth flush io 60s ident 5m mbuff 8k rbuff 4k rnums 3 window 10s dest files io info user redir xrootd.t2.ucsd.edu:9930 dest files iov info user xrootd.t2.ucsd.edu:9932
+
+xrd.network keepalive kaparms 10m,1m,5
+xrd.timeout idle 60m
+```
+
+### Authorization file
+
+For the authorization options on XRootD please see [here](/data/install-xrootd/#optional-authorization)
+
+An example of an authorization file for read only of store by CMS would be:
+
+```file
+g /cms /store lr
+```
+
+#### LCMAPS authorization
+
+To configure the LCMAPS authentication please follow the documentation
+[here](/data/install-xrootd/#security-option-3-xrootd-lcmaps-authorization)
+
+### Optional Configuration
+
+The following configuration steps are optional and will likely not be required for setting up a small site.
+If you do not need any of the following special configurations, skip to
+[the section on using XRootD](#using-xrootd).
+
+#### Enabling HTTP support
+
+In order to enable XRootD HTTP support please follow the instructions
+[here](/data/install-xrootd/#optional-enabling-xrootd-over-http)
+
+#### Enabling Hadoop support
+
+For documentation on how to export your Hadoop storage using XRootD please see
+[this documentation](/data/install-xrootd/#optional-adding-hadoop-support-to-xrootd)
+
+#### Enabling CMS TFC support (CMS sites only)
+
+For CMS users, there is a package available to integrate rule-based name lookup using a `storage.xml` file.
+See [this documentation](/data/install-xrootd/#optional-adding-cms-tfc-support-to-xrootd-cms-sites-only).
+
+Using XRootD
+------------
+In order to start your XRootD server you need to run
+
+``` console
+root@xrootd-server # systemctl start xrootd@%RED%server%ENDCOLOR%
+```
```

Comment:
What's an appropriate value for `server` here? The systemctl command seems superfluous with the table that follows. Let's tell them how to construct `SERVICE-NAME` and then they can use the table to sort out service management themselves.